store/postgres/src/relational/index.rs

```diff
@@ -411,6 +427,32 @@ impl CreateIndex {
         }
     }
 
+    pub fn with_nsp(&self, nsp2: String) -> Self {
+        let s = self.clone();
+        match s {
+            CreateIndex::Unknown { defn } => CreateIndex::Unknown { defn },
```

Comment:
This is a tough one: since we failed to parse the create index statement, we have no idea what's in it, and I think `with_nsp` should return an error. The danger is that `defn` will talk about some namespace, but we will then run it thinking we changed the namespace, which will be sure to produce the wrong result.

Reply:
Right. Added an error result here.